src/main/java/org/folio/circulation/support/CirculationRulesClient.java

```diff
@@ -44,10 +45,11 @@ private String queryParameters(
     String loanTypeId,
     String locationId,
     String materialTypeId,
+    String institutionId,
```

Comment:
Can you replace the locationId and institutionId parameters by a single domain/Location.java parameter?
domain/Location.java contains locationId, institutionId, campusId and libraryId that are needed for CIRC-413 and CIRC-414.

Reply:
Fixed